src/entry.rs

```diff
@@ -769,6 +769,75 @@ entry! {
     }
 }
 
+entry! {
+    pub fn _SSL_CTX_add_server_custom_ext(
+        ctx: *mut SSL_CTX,
+        ext_type: c_uint,
+        _add_cb: custom_ext_add_cb,
+        _free_cb: custom_ext_free_cb,
+        _add_arg: *mut c_void,
+        _parse_cb: custom_ext_parse_cb,
+        _parse_arg: *mut c_void,
+    ) -> c_int {
+        try_clone_arc!(ctx);
```

Comment:
Is this `try_clone_arc!(ctx);` here just to reject `ctx == NULL`? Maybe worth a comment if so since it seems a bit odd to clone the arc without using the result for anything.